tests/copy-paste.js

```diff
@@ -444,15 +444,15 @@ define([
             util.loadXML("");
             paste([
                 ["id", "type", "calculateAttr", "conflictedNodeId"],
-                ["/radius", "DataBindOnly", "42", "null"],
```

Comment:
Copy/paste never actually used this format as JSON.parse would parse this into the number 42, not a string. Which means that if you opened the browser on this test you could never select radius or pi questions.